src/components/AvatarGroup/AvatarGroup.jsx

```diff
@@ -38,13 +43,15 @@ const AvatarGroup = ({
           ...avatar?.props,
           size: size || avatar?.props?.size,
           type: type || avatar?.props?.type,
-          className: cx(styles.avatarContainer, avatarClassName)
+          className: cx(styles.avatarContainer, avatarClassName),
+          onClick: () => avatarOnClick(avatar)
```

Comment:
Not sure we should pass back all the props, but we should pass some props so user would be able to comfortably determine which avatar was clicked and work with it afterwards. Maybe we should send back only `id`? Perhaps `ariaLabel`, `text` for some cases? And maybe `index`? What do you think?
I know we can send back just the event, but for MenuItem it will be a different type of event, which user would have to handle specifically, which I guess we don't want to - I want to pass exactly same arguments whether Avatar was clicked, avatar from the AvatarGroup or MenuItem with avatar inside

Reply:
it should be the avatarId